Mapping/ContainerAwareEntityListenerResolver.php

```diff
@@ -2,107 +2,16 @@
 
 namespace Doctrine\Bundle\DoctrineBundle\Mapping;
 
-use InvalidArgumentException;
-use RuntimeException;
 use Symfony\Component\DependencyInjection\ContainerInterface;
 
-class ContainerAwareEntityListenerResolver implements EntityListenerServiceResolver
+/**
+ * @deprecated since 1.11 and will be removed in 2.0. Use ContainerEntityListenerResolver instead.
```

Comment:
since *Doctrine* [...]
so that ppl can figure out their deprecation logs more easily (should be a policy really :) )